yarn-project/acir-simulator/src/acvm/acvm.ts

```diff
@@ -2,12 +2,12 @@ import { AztecAddress } from '@aztec/foundation/aztec-address';
 import { EthAddress } from '@aztec/foundation/eth-address';
 import { Fr } from '@aztec/foundation/fields';
 import { createDebugLogger } from '@aztec/foundation/log';
-import { solve_intermediate_witness as solveIntermediateWitness } from '@noir-lang/aztec_backend_wasm';
+import { executeCircuit } from 'acvm-simulator';
 
 /**
  * The format for fields on the ACVM.
  */
-export type ACVMField = `0x${string}`;
+export type ACVMField = string;
```

Comment:
acvm-simulator now has type definitions for fields and witnesses, so we update them here for them to match.